cobalt/tools/on_device_tests_gateway_client.py

```diff
@@ -294,13 +294,13 @@ def main() -> int:
   trigger_parser.add_argument(
       '--job_timeout_sec',
       type=str,
-      default='1800',
+      default='3600',
```

Comment:
We don't want to increase this timeout.

If blink_unittests takes this long to finish it can't be run as-is on presubmit.

Our current longest running test takes ~12 minutes end-to-end. We don't want it to take significantly longer than that.